zammad/templates/statefulset.yaml

```diff
@@ -42,24 +42,27 @@ spec:
         {{- with .Values.initContainers }}
         {{- toYaml . | nindent 8}}
         {{- end }}
-        {{- if .Values.volumePermissions.enabled }}
-        - name: data-chmod
-          image: "{{ .Values.volumePermissions.image.repository }}:{{ .Values.volumePermissions.image.tag }}"
-          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy }}
+        {{- if .Values.zammadConfig.initContainers.volumePermissions.enabled }}
+        - name: zammad-volume-permissions
+          image: "{{ .Values.zammadConfig.initContainers.volumePermissions.image.repository }}:{{ .Values.zammadConfig.initContainers.volumePermissions.image.tag }}"
+          imagePullPolicy: {{ .Values.zammadConfig.initContainers.volumePermissions.image.pullPolicy }}
           command:
             - /bin/sh
             - -cx
             - |
-              chown {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} -R /opt/zammad
+              chown {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} -R /opt/zammad/var
+              chmod 770 /opt/zammad/tmp
```

Comment:
Are you sure about setting this via chmod here? If somebody disables volumePermissions, then /opt/zammad/tmp will have 777 permission and uploads will start failing.
My solution with in-memory storage would avoid this issue because it has permission 1777 which is acceptable for Ruby.

Reply:
Ok, thanks.